app/assets/stylesheets/forever_style_guide/modules/_list.scss

```diff
@@ -48,6 +48,52 @@
   }
 }
 
+.list-group-toggle {
+  .badge {
+    @extend %check-style;
+    padding: 5px 0 0;
+    font-size: font-size('default');
+    color: color('gray-500');
+    @include circle(24px, color('gray-200'));
+
+    &:hover {
+      color: color('primary');
```

Comment:
primary green seems like an odd choice here, we use secondary/blue for almost all our other hover link states.